effekt/shared/src/main/scala/effekt/Namer.scala

```diff
@@ -291,6 +303,17 @@ object Namer extends Phase[Parsed, NameResolved] {
       }
 
     case f @ source.ExternDef(capture, id, tparams, vparams, bparams, ret, bodies) =>
+    //      if (!bodies.supportedByFeatureFlags(Context.compiler.supportedFeatureFlags)) {
+    //        val featureFlags = bodies.map(_.featureFlag)
+    //        //        Context.warning(pp"Extern definition ${id} is not supported as it is only defined for feature flags ${featureFlags.mkString(", ")}," +
+    //        //          pp"but the current backend only supports ${Context.compiler.supportedFeatureFlags.mkString(", ")}.")
+    //      }
+    //      bodies.foreach {
+    //        case source.ExternBody.StringExternBody(ff, _) if ff.isDefault =>
+    //          Context.warning(pp"Extern definition ${id} contains extern string without feature flag. This will likely not work in other backends, "
+    //            + pp"please annotate it with a feature flag (Supported by the current backend: ${Context.compiler.supportedFeatureFlags.mkString(", ")})")
+    //        case _ => ()
+    //      }
```

Comment:
@marzipankaiser I have this commented out here; don't we have your check on master anymore? The diff suggests this

Reply:
I'll go ahead and delete this to minify the diff; please complain if we still need this.